icarus_backend/mission/MissionViews.py

```diff
@@ -0,0 +1,27 @@
+from django.http import HttpResponse
+from django.utils import timezone
+import json
+from django.views.decorators.csrf import csrf_exempt
+import uuid
+from django.utils.dateparse import parse_datetime
+from django.contrib.gis.geos import Polygon
+from .MissionModel import Mission
+
+@csrf_exempt
```

Comment:
yeah this was the way that I was disabling them

Reply:
I forgot to get rid of this one, are you cool with doing csrf exemption through the middleware?